pygfx/controllers/_panzoom.py

```diff
@@ -72,41 +108,85 @@ def look_at(self, eye: Vector3, target: Vector3, up: Vector3) -> Controller:
         self.rotation.set_from_rotation_matrix(self._m.look_at(eye, target, up))
         return self
 
-    def pan(self, vec3: Vector3) -> Controller:
-        """Pan in 3D world coordinates."""
+    def pan(self, vec3: Vector3, button: int) -> Controller:
+        """Pan or stretch in 3D world coordinates."""
         self.target.add(vec3)
+
+        if button == 1:
+            pass
+
+        elif button == 2:
```

Comment:
Instead of this triage, the code would be more readable (I think) if it had a separate `def stretch()` for the new functionality. Same for start and stop.

Reply:
I agree